com.unity.render-pipelines.high-definition/Editor/Lighting/HDLightEditor.cs

```diff
@@ -72,15 +73,22 @@ void OnUndoRedo()
         public override void OnInspectorGUI()
         {
             m_SerializedHDLight.Update();
-
             // Add space before the first collapsible area
             EditorGUILayout.Space();
 
             ApplyAdditionalComponentsVisibility(true);
 
             EditorGUI.BeginChangeCheck();
-            using (new EditorGUILayout.VerticalScope())
-                HDLightUI.Inspector.Draw(m_SerializedHDLight, this);
+
+            if (HDEditorUtils.IsPresetEditor(this))
+            {
+                HDLightUIPreset.Inspector.Draw(m_SerializedHDLight, this);
+            }
+            else
+            {
+                using (new EditorGUILayout.VerticalScope())
+                    HDLightUI.Inspector.Draw(m_SerializedHDLight, this);
+            }
```

Comment:
Now that I see this it will be nice to have something like `HDEditorUtils.DrawPresetEditor(HDLightUIPreset.Inspector, HDLightUI.Inspector, m_SerializedHDLight, this)` as this else if statement is in multiple places. Internally the method will decide what to draw

Reply:
I dont think having Preset editors are a great pattern and this is a temporary fix until we can land proper preset support.

Reply:
perfect then :)